rmw_fastrtps_cpp/package.xml

```diff
@@ -20,8 +20,8 @@
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
 
-  <build_depend>fastcdr</build_depend>
-  <build_depend>fastrtps</build_depend>
+  <build_depend version_gte="2">fastcdr</build_depend>
+  <build_depend version_gte="2.13">fastrtps</build_depend>
```

Comment:
Let's not add these constraints in here.  I know it is technically more correct, but we don't do this anywhere else.

(same goes for all of the uses below)

Reply:
This is something we mentioned within the Production WG while discussing [this proposal](https://github.com/bpwilcox/ros2-production-working-group/issues/11)

Not doing it anywhere else doesn't mean it should not be done.
Although it seems that [rosdep will not enforce version checks](https://github.com/ros-infrastructure/rosdep/issues/325),  bloom will put the version requirements in the rules to build debian packages.

That said, I will add a commit removing version restrictions from all `package.xml` files, and add them in a follow-up PR so we don't block merging this one

Reply:
3d2f5f5

Reply:
> Although it seems that [rosdep will not enforce version checks](https://github.com/ros-infrastructure/rosdep/issues/325), bloom will put the version requirements in the rules to build debian packages.

It's true that bloom will put those version checks in, but there isn't really a scenario where it would ever be needed, especially in the core and in Rolling.  Releasing this package will cause all of the downstream packages to rebuild (which is most of the ROS ecosystem).  If any of that fails to build, then we won't have a viable distribution to "sync".  If it all succeeds in building, then we will sync the whole thing out together.  In that sense, it is "atomic" with the rest of the packages (at least up to `desktop`).  You can definitely construct scenarios where someone cherry-picks certain packages and tries to install them together, but anyone doing that on Rolling gets to keep all of the pieces, in my opinion.

Anyway, I agree with that this is a wider discussion we should have, so removing it from this PR is the correct thing for now.